src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/writefinaldecision/uc/UcNewTemplateContent.java

```diff
@@ -1,15 +1,35 @@
 package uk.gov.hmcts.reform.sscs.ccd.presubmit.writefinaldecision.uc;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
 import java.util.Arrays;
 import java.util.List;
 import org.apache.commons.lang3.StringUtils;
+import uk.gov.hmcts.reform.sscs.ccd.presubmit.writefinaldecision.uc.scenarios.UcScenario;
 import uk.gov.hmcts.reform.sscs.ccd.presubmit.writefinaldecision.uc.scenarios.UcTemplateComponentId;
 import uk.gov.hmcts.reform.sscs.model.docassembly.Descriptor;
 import uk.gov.hmcts.reform.sscs.model.docassembly.Paragraph;
 import uk.gov.hmcts.reform.sscs.model.docassembly.WriteFinalDecisionTemplateBody;
 
 public abstract class UcNewTemplateContent extends UcTemplateContent {
 
+    protected static DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+
+    public abstract UcScenario getScenario();
+
+    public String getAllowedOrRefusedSentence(boolean allowed) {
```

Comment:
I'm sure this method exists in UcTemplateContent. I'm not sure if Michael's intent is to remove UcTemplateContent, if that is the case it's fine to copy it here.

Reply:
Yes, He renamed it to UcOldTemplateContent and eventually be removed.